Makefile

```diff
@@ -21,10 +21,6 @@ v: unit_test
 #	valgrind -q --leak-check=full ./v7 tests/run_tests.js
 #	gcov -a unit_test.c
 
-xrun: unit_test
-	$(CC) -W -Wall -I. -I./src src/tokenizer.c -DTEST_RUN -DV7_EXPOSE_PRIVATE -o t
-	./t
-
 run:
 	@$(MAKE) -C tests coverage
 
```

Comment:
Reverted that change, it doesn't belong to this PR.
TBD in separate PR.
This is an old, outdated tokenizer "test" that should be removed from the makefile and tokenizer.c.